x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/entsearch/analytics/behavioral_analytics-events-mappings.json

```diff
@@ -86,16 +86,7 @@
                 "ignore_above": 1024
             },
             "filters": {
-              "properties": {
-                "field": {
-                    "type": "keyword",
-                    "ignore_above": 1024
-                },
-                "value": {
-                    "type": "keyword",
-                    "ignore_above": 1024
-                }
-              }
+              "type": "flattened"
```

Comment:
Looks fine to use `flattened` for now but that reminds me that we should discuss the mapping a bit more. We have a lot of `ignore_above` on keyword fields so I wonder what's the expectations on these fields.